datetime/src/commonMain/kotlin/DateTime.kt

```diff
@@ -0,0 +1,67 @@
+/*
+ Copyright 2020 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.datetime
+
+import com.splendo.kaluga.datetime.datecomponents.Day
+import com.splendo.kaluga.datetime.datecomponents.Month
+import com.splendo.kaluga.datetime.datecomponents.Year
+
+data class DateTime(
+    val timestamp: Double
+) {
+    companion object {
+        fun now(): DateTime {
+            val millis = getTimeMillis()
+            return DateTime(millis.toDouble())
+        }
+    }
+
+    override fun equals(other: Any?): Boolean =
+        if (other is DateTime) timestamp == other.timestamp
+        else false
+}
+
+fun DateTime.millisUntil(dateTime: DateTime): Long =
```

Comment:
Lets also add a before/after convenience method

Reply:
Good idea. (until now I added mostly methods we needed in WTS)